desktop/apps/auction2/actions.js

```diff
@@ -2,14 +2,17 @@ import metaphysics from '../../../lib/metaphysics.coffee'
 import { filterQuery } from './filter_query'
 
 // Action types
+export const TOGGLE_FETCHING_ARTWORKS = 'TOGGLE_FETCHING_ARTWORKS'
```

Comment:
Ah! I did see some places suggesting to do that. I guess it makes more sense to have the action types be self-documenting, rather than the contents of the payload for something like this.